WinUIGallery/ControlPages/DesignGuidance/AccessibilityKeyboardPage.xaml

```diff
@@ -1,387 +1,673 @@
+
+                            Note that:
+
+                                - The buttons are still tabbable by default
+                                - Home/End and PgUp/PgDn do not work
+                                - Screen readers do not read PositionInSet/SizeOfSet ("1 of 3")
+
+                            All of that would require custom work.
+                        -->
+                        <StackPanel
+                            Spacing="4"
+                            Orientation="Horizontal"
+                            XYFocusKeyboardNavigation="Enabled">
+                            <Button Content="Boil 'em"  />
+                            <Button Content="Mash 'em" />
+                            <Button Content="Stick 'em in a stew" />
+                        </StackPanel>
+                    </StackPanel>
+                    <TextBlock
+                        Style="{ThemeResource CaptionTextBlockStyle}"
+                        Text="Arrow keys navigate between each button, but tab still navigates between each button, and other buttons like Home/End do not work" />
+                </StackPanel>
+            </core:ControlExample.Example>
+            <core:ControlExample.Xaml>
+                <x:String xml:space="preserve">
+&lt;StackPanel
+    Background="{ThemeResource CardBackgroundFillColorSecondaryBrush}"
+    BorderBrush="{ThemeResource SurfaceStrokeColorDefaultBrush}"
+    AutomationProperties.Name="Potatoes?"
+    BorderThickness="1"
+    Spacing="8"
+    Padding="8"
+    CornerRadius="4"&gt;
+    &lt;TextBlock
+        Style="{ThemeResource BodyStrongTextBlockStyle}"
+        Text="Potatoes?" /&gt;
+    &lt;!--
+        XYFocusKeyboardNavigation enables arrow keys between the buttons.
+
+        Note that:
+
+            - The buttons are still tabbable by default
+            - Home/End and PgUp/PgDn do not work
+            - Screen readers do not read PositionInSet/SizeOfSet ("1 of 3")
+
+        All of that would require custom work.
+    --&gt;
+    &lt;StackPanel
+        Spacing="4"
+        Orientation="Horizontal"
+        XYFocusKeyboardNavigation="Enabled"&gt;
+        &lt;Button Content="Boil 'em"  /&gt;
+        &lt;Button Content="Mash 'em" /&gt;
+        &lt;Button Content="Stick 'em in a stew" /&gt;
+    &lt;/StackPanel&gt;
+&lt;/StackPanel&gt;
+                </x:String>
+            </core:ControlExample.Xaml>
+        </core:ControlExample>
+        
+        <TextBlock
+            Margin="0,20,0,0"
+            Style="{ThemeResource SubtitleTextBlockStyle}"
+            AutomationProperties.HeadingLevel="Level2"
+            Text="Keyboard shortcuts" />
+
+        <RichTextBlock>
+            <Paragraph>
+                Keyboard shortcuts are extremely helpful for Narrator users, keyboard users, and power users.
+                Since these groups often find it difficult to quickly jump between sections of UI like mouse users,

```

Comment:
Maybe not focus on the people using on keyboard shortcuts but rather on the that keyboard navigation generally lacks the ability to switch between sections?

Reply:
I think that's a good inclusive language choice!